### ISSUE TITLE
AssemblyAI - Deep Learning Researcher (ASR) - Remote/Global

### ISSUE THREAD

**Issue by kbassemblyai** [TechnologyMasters/jobs#440]
### Salary Expectation

- $140,000 - $200,000+ USD per year (or equivalent)
- Full-Time

### Benefits

- Competitive Salary + Bonus
- Platinum Benefits
- Unlimited PTO
- Equity
- Fully Remote
- $1k budget for home office equipment

### Location

- 100% Remote Global Team - Full List [Here](https://help.letsdeel.com/en/articles/4759216-in-which-countries-is-deel-available)

## What You'll Do 

Our ASR models outperform big tech companies like Google, AWS, and Microsoft. That being said, there's still a lot of work to do to match human level accuracy. Our Deep Learning team is a tight knit group of creative researchers and engineers, who are not afraid to try unconventional ideas. In this role, you'll:

- Work with large scale datasets to research and train Deep Learning models for Speech Recognition
- Conduct research and experiments in order to improve accuracy of Deep Learning ASR pipelines like CTC, LAS and RNN-Ts
- Dig into weaknesses and failure points of our current ASR models, in order to identify further areas for improvement
- Work with the broader Speech Recognition team to publish papers on novel findings
- Continually push the State of the Art in Speech Recognition to get to human level performance

### Must Have

- 2+ years of Deep Learning experience
- 2+ years experience with PyTorch or TensorFlow
- 2+ years experience with C++
- Experience with CUDA
- Experience training large models on multiple GPUs

### Nice to Have

- Experience with ASR (can be experience from a PhD, Masters, or work experience)
- Experience with fully Deep Learning based ASR systems (CTC, RNNT, Wav2Letter)

### Relevant Experiences

- Deep Learning
- ASR (Automatic Speech Recognition)

## About Our company

AssemblyAI (www.assemblyai.com) is the #1 rated API for Speech Recognition. Thousands of developers use our APIs to transcribe millions of videos, podcasts, phone calls, and zoom meetings every day - powering innovative products like visual voicemail, meeting summarizers, and closed captioning.

We're backed by leading investors including Y Combinator, John and Patrick Collison of Stripe, Daniel Gross, and Nat Friedman. We were founded in 2017.

## How to apply

[Apply Directly Here 
](https://apply.workable.com/assemblyai/j/732C9A8A7F/)---

## Meta

- [X] Full Time
- [ ] Part Time
- [ ] Contract
- [ ] Internship
- [X] Remote Worldwide
- [ ] Remote Regional
- [X] Remote OK

**Comment by SuperStar0225** [comment]
I am interest in this job.
